src/refda/amm/sbr.c

```diff
@@ -40,3 +41,11 @@ void refda_amm_sbr_desc_deinit(refda_amm_sbr_desc_t *obj)
     cace_ari_deinit(&(obj->min_interval));
     memset(obj, 0, sizeof(*obj));
 }
+
+bool refda_amm_sbr_desc_reached_max_exec_count(refda_amm_sbr_desc_t *obj)
+{
```

Comment:
Public functions need to guard their inputs, just a simple `CHKFALSE(obj)` would work.

Reply:
Good catch, thanks!